src/config.rs

```diff
@@ -48,6 +48,14 @@ pub struct Config {
     pub tokio_console_enabled: bool,
     /// See [`TokioConsoleConfig::listen_address`].
     pub tokio_console_listen_address: SocketAddr,
+    /// Enables refreshing upload metrics from Janus. Enabled by default.
+    pub enable_metrics_refresh: bool,
```

Comment:
The smallest concern: two lines above we have tokio_console_enabled, but this is enable_. Ideally we'd be consistent about predicate env var naming, but this isn't at all a blocker